drools-wb-screens/drools-wb-scenario-simulation-editor/drools-wb-scenario-simulation-editor-client/src/main/resources/org/drools/workbench/screens/scenariosimulation/client/resources/i18n/ScenarioSimulationEditorConstants.properties

```diff
@@ -27,6 +27,49 @@ hideRightPanel=Hide Right Panel
 testTools=Test Tools
 testEditor=Test Editor
 scenarioCheatSheet=Scenario Cheatsheet
+dmnCheatSheet=<p>\
```

Comment:
Do we need this tag? I'm not sure, but I think it causes an unnecessary vertical space.

![bug1](https://user-images.githubusercontent.com/16349113/52043137-2d9de280-253f-11e9-9469-45422a4b0ca0.png)

Reply:
I've removed it :)

Reply:
Also modified "cheatsheet" to be more consistent (cheatSheet / *CheatSheet)